makefile

```diff
@@ -47,6 +49,15 @@ consul:
 # build and release
 
 # build our binary in a container
+
+ifeq "$(TRAVIS)" "true"
```

Comment:
In my testing, there was a permissions problem with chmod after it builds the binary with docker.

Also, I was attempting to use the native go binary on the travis CI machine, since it allows us to build it against different go versions